spec/std/dir_spec.cr

```diff
@@ -274,7 +274,7 @@ describe "Dir" do
       ].sort
     end
 
-    it "matches symlinks" do
+    pending_win32 "matches symlinks" do
```

Comment:
Why/how does this fail and can it be fixed?

Reply:
The first `File.symlink` fails with `WinError 1314 A required privilege is not held by the client`.

It seems [`CreateSymbolicLinkW`](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createsymboliclinkw) requires admin rights.

Reply:
Can we design a spec helper which detects admin rights and marks the spec pending depending?

Reply:
Sure. But this can be postponed. It's marked as pending, so it naturally means we need to come back to this.

Reply:
Well judging by the number of pending specs which have been hanging around for years...